src/app/pages/information/information.page.scss

```diff
@@ -0,0 +1,12 @@
+ion-slides {
+    width: 100%;
+  }
+  
+  .slide-card {
+    width: 100%;
+    margin-bottom: 35px;
+  }
+  
+  .multiline {
+    white-space: pre-wrap;
+  }
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6012707)